app/Http/Controllers/Api/WorkController.php

```diff
@@ -48,6 +47,8 @@ private function loadWorks(Request $request)
     {
         $works = Work::query();
 
+        $works->with('media')->with('other_architects');
```

Comment:
```suggestion
        $works->with(['media', 'other_architects']);
```
https://laravel.com/docs/8.x/eloquent-relationships#eager-loading-multiple-relationships